stream-chat-android-client/src/main/java/io/getstream/chat/android/client/channel/ChannelClient.kt

```diff
@@ -654,32 +653,22 @@ public class ChannelClient internal constructor(
 
     @CheckResult
     public fun keystroke(): Call<ChatEvent> {
```

Comment:
Can we remove this method? We have the same one below 🤔

Reply:
The other one takes `parentId`. Then we will have to add `@JvmOverloads`.

Reply:
Let's keep only one and add `@JvmOverloads`